modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -249,9 +249,9 @@ export async function getPickList(pickListId: number): Promise<PickList> {
 	);
 }
 
-export async function getPickListItems(pickListId: number) {
+export async function getPickListItems(externalReferenceCode: string) {
```

Comment:
Was this change intentional? I don't remember changing this part when I did the commit, so maybe the branch wasn't updated when was pushed causing this? Can you please check if this should be pickListId or externalReferenceCode?

Also please check the rest of this commit if things as they should be comparing with your local, thanks!